redash/query_runner/salesforce.py

```diff
@@ -159,7 +159,7 @@ def run_query(self, query, user):
             data = {'columns': columns, 'rows': rows}
             json_data = json_dumps(data)
         except SalesforceError as err:
-            error = err.message
+            error = err.content
```

Comment:
How about we take it a step forward and make it really friendly with something like: 

```python
if isinstance(err.content, list):
    error = "\n".join([content.get('message', '') for content in err.content)
else:
    error = err.content
```

?